src/software/ai/evaluation/possession.h

```diff
@@ -1,7 +1,67 @@
 #pragma once
 
+#include "software/time/duration.h"
+#include "software/time/timestamp.h"
 #include "software/world/world.h"
 
+// Indicates which team has possession of the ball
+enum TeamPossession
+{
+    FRIENDLY_TEAM,
+    ENEMY_TEAM
+};
```

Comment:
I was experimenting with having NO_TEAM to indicate no team has clear possession over the ball and different degrees of possession (e.g. AGGRESSIVE_ENEMY_TEAM), figured it would be easier to make changes if we end up returning to those ideas

Reply:
Ok that makes more sense now